packages/hoprd/package.json

```diff
@@ -75,8 +75,7 @@
     "require": "ts-node/register"
   },
   "hopr": {
-    "description": "master",
-    "environment_id": "master"
+    "description": "master"
```

Comment:
Why is this still needed and why is it under a `hopr` key when it regards `hoprd`?